lib/lib.cpp

```diff
@@ -0,0 +1,246 @@
+#include <iostream>
+#include <string>
+#include <stack>
+#include <vector>
+#include <algorithm>
+#include <cctype>
+
+#include "lib.h"
+
+int priority(const std::string& current_operator)
+{
+    if (current_operator == "*" || current_operator == "/" || current_operator == "%")
+        return 2;
+    if (current_operator == "+" || current_operator == "-")
+        return 1;
+    return 0;
+}
+
+bool is_operator(char symbol)
+{
+    std::string symbol_str;
+    symbol_str.push_back(symbol);
```

Comment:
you could just write `std::string symbol_str = symbol`

Reply:
++